<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find multiAgent_system_modeling)/config/irobot_control.yaml" command="load"/>

  <!-- load the controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
	output="screen" ns="/robot1" args="joint_state_controller wheels_position_controller"/>

  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
  output="screen" ns="/robot2" args="joint_state_controller wheels_position_controller"/>

  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
  output="screen" ns="/robot3" args="joint_state_controller wheels_position_controller"/>

  <!-- convert joint states to TF transforms for rviz, etc -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
	respawn="false" output="screen">
    <!-- <remap from="/joint_states" to="/irobot/joint_states" /> -->
  </node>

</launch>